### PR TITLE
Bump 'Tested up to' version to 6.3

### PR DESCRIPTION
**Pull request by kienstra**
* Bump 'Tested up to' version to 6.3
* To deploy to SVN, approve the CircleCI job

### CODE REVIEW

**Comment by kienstra** [comment]
This [succeeded](https://plugins.trac.wordpress.org/changeset?old=2892978&old_path=genesis-responsive-slider&new=2948742&new_path=genesis-responsive-slider).

**Comment by kienstra** [comment]
Hi @dreamwhisper,
This is one of the few ones that requires an approval before merging 😄

**Review by dreamwhisper — APPROVED** [review]

**Comment by kienstra** [comment]
Thanks a lot, @dreamwhisper!

**Pull request merged**
